apps/docs/docs/template/api/experimental/rx-if-directive.md

```diff
@@ -1,3 +1,3 @@
-# 🧪 RxIf Directive
+# 🧪 RxIf directive
```

Comment:
Should we unify the names? 

Example:
- RxIf directive
- LetDirective

Reply:
The source code isn't consistent
- https://github.com/rx-angular/rx-angular/blob/b779f885f066f6caad465086d5100ab3d697f91a/libs/template/experimental/if/src/lib/if.directive.ts#L42
- https://github.com/rx-angular/rx-angular/blob/b779f885f066f6caad465086d5100ab3d697f91a/libs/template/for/src/lib/for.directive.ts#L231
- https://github.com/rx-angular/rx-angular/blob/b779f885f066f6caad465086d5100ab3d697f91a/libs/template/let/src/lib/let.directive.ts#L179
- https://github.com/rx-angular/rx-angular/blob/b779f885f066f6caad465086d5100ab3d697f91a/libs/template/experimental/viewport-prio/src/lib/viewport-prio.directive.ts#L64